.github/workflows/installer-tests.yml

```diff
@@ -0,0 +1,47 @@
+name: Tests for ansible-rulebook installer
+
+on:
+  pull_request:
+    branches: ["main"]
+    paths:
+      - 'roles/install_ansible_rulebook/**'
+      - 'playbooks/install_rulebook_cli.yml'
+
+jobs:
+  cli-installer-tests:
+    strategy:
+      matrix:
+        os:
+          - macos-latest
+        include:
+          - os: ubuntu-latest
```

Comment:
wrong value

Reply:
The `include`? I got that from the [matrix config docs](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#expanding-or-adding-matrix-configurations). It does work, but maybe theres a better way to do it?

Reply:
is not ubuntu, but fedora

Reply:
nevermind, my mistake